src/Components/Navbar/Navbar.jsx

```diff
@@ -19,12 +22,14 @@ const Navbar = () => {
             <span>2</span>
           </div>
         </div>
-        <div className="heart-badge">
-          <i className="fas fa-shopping-cart fa-2x"></i>
-          <div className="notification-badge flex-row">
-            <span>4</span>
+        <Link className="link" to="/cart">
+          <div className="heart-badge">
+            <i className="fas fa-shopping-cart fa-2x"></i>
+            <div className="notification-badge flex-row">
+              <span>{cartState.cart.length}</span>
```

Comment:
Just a suggestion : you could hide the badge if there is no item in cart and show only when something is added.

Reply:
Thanks for the suggestion, I have done the changes.